runtime/interpreter/value.go

```diff
@@ -1364,6 +1447,12 @@ func (Int16Value) SetMember(_ *Interpreter, _ LocationRange, _ string, _ Value)
 	panic(errors.NewUnreachableError())
 }
 
+func (v Int16Value) ToBigEndianBytes() []byte {
+	b := make([]byte, 2)
+	binary.BigEndian.PutUint16(b, uint16(v))
```

Comment:
I just realized this, but if we encode arbitrary-length integers as their two's complement, do you think it makes sense to also do it for all integers?

Reply:
Yep, negative built-in integers are already represented in two's complement, so nothing to be done here. See also the tests: https://github.com/onflow/cadence/pull/206/files#diff-b17c665b13703eae54b6cea9fee6aa63R142-R144

Reply:
Ah I see! Looks good 👍